core/src/main/scala/io/chrisdavenport/github/endpoints/Search.scala

```diff
@@ -0,0 +1,32 @@
+package io.chrisdavenport.github.endpoints
+
+import cats.data._
+import cats.effect._
+import io.chrisdavenport.github.data.Repositories._
+import org.http4s._
+import org.http4s.implicits._
+import org.http4s.client.Client
+import io.chrisdavenport.github.Auth
+import io.chrisdavenport.github.data.SearchResult
+import io.chrisdavenport.github.data.SearchResult.{Order, Sort}
+import io.chrisdavenport.github.internals.GithubMedia._
+import io.chrisdavenport.github.internals.RequestConstructor
+
+object Search {
+
+  def repository[F[_]: Sync](
```

Comment:
Can you link to the appropriate endpoint documentation for easy debugging in the future?

Reply:
Yep, will do!